cmd/otel-agent/config/agent_config_test.go

```diff
@@ -69,7 +71,7 @@ func (suite *ConfigTestSuite) TestAgentConfigDefaults() {
 	assert.Equal(t, "datadoghq.com", c.Get("site"))
 	assert.Equal(t, "https://api.datadoghq.com", c.Get("dd_url"))
 	assert.Equal(t, true, c.Get("logs_enabled"))
-	assert.Equal(t, "https://agent-http-intake.logs.datadoghq.com", c.Get("logs_config.logs_dd_url"))
+	assert.Equal(t, "https://http-intake.logs.datadoghq.com", c.Get("logs_config.logs_dd_url"))
```

Comment:
Lets not change the default logs URL

Reply:
This is tricky, because the upstream datadogconfig sets the logs endpoint to `https://http-intake.logs.{DD_SITE}` in `Unmarshal`  https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/datadog/config/config.go#L263

We can change the endpoint used by logs agent in agent_config to  `https://agent-http-intake.logs.{DD_SITE}` but the dd flare extension still displays `https://http-intake.logs.{DD_SITE}` (because it is set during unmarshal confmap) WDYT